### DISCUSSION TITLE
查询该歌词源时发生错误：ReferenceError：structuredClone is not defined

### DISCUSSION THREAD

**Discussion by huoyan1231** [amll-dev/applemusic-like-lyrics#224] [Q&A]
![image](https://github.com/Steve-xmh/applemusic-like-lyrics/assets/53113734/309cc34a-faef-400b-b5ca-d6ffed50f364)

**Comment by huoyan1231** [comment]
只有Dev分支的测试版有此问题

**Comment by ITManCHINA** [comment]
您好，请使用3.1.0插件最终版发布版本
